pyisy/events.py

```diff
@@ -240,12 +239,13 @@ def watch(self):
                 return
 
             for message in events:
-                try:
-                    self._route_message(message)
-                except Exception as ex:  # pylint: disable=broad-except
-                    self.isy.log.warning(
-                        "PyISY encountered while routing message '%s': %s", message, ex
-                    )
+                # try:
+                #     self._route_message(message)
+                # except Exception as ex:  # pylint: disable=broad-except
+                #     self.isy.log.warning(
+                #         "PyISY encountered while routing message '%s': %s", message, ex
+                #     )
```

Comment:
Bah! Good catch. Nooope that definitely needs to be there. I need to fix my debug path.

Reply:
Side note -- can you please test the new ISY code on HA dev if you get a chance--I want to make sure I didn't break your old heartbeat fixes--the only significant changes should be 1) to assume it's OK for the first heartbeat period (because you may be restarting HA often and they never actually make it 24 hrs) and 2) respond to both `DON` and `DOF` commands on the heartbeat since the device toggles back and forth.

Reply:
Sure, I'll get it pulled down!